docs/README.md

```diff
@@ -21,30 +21,30 @@
  Tips  Find more information at: https://kubecm.cloud
 
 Usage:
-  kubecm [command]
+  kubecm(kc) [command]
 
 Available Commands:
-  add         Add KubeConfig to $HOME/.kube/config
-  alias       Generate alias for all contexts
-  clear       Clear lapsed context, cluster and user
-  cloud       manage kubeconfig from cloud
-  completion  Generate completion script
-  create      Create new KubeConfig(experiment)
-  delete      Delete the specified context from the kubeconfig
-  help        Help about any command
-  list        List KubeConfig
-  merge       Merge multiple kubeconfig files into one
-  namespace   Switch or change namespace interactively
-  rename      Rename the contexts of kubeconfig
-  switch      Switch Kube Context interactively
-  version     Print version info
+  add(a)         Add KubeConfig to $HOME/.kube/config
+  alias(al)      Generate alias for all contexts
+  clear          Clear lapsed context, cluster and user
+  cloud          manage kubeconfig from cloud
+  completion(c)  Generate completion script
+  create         Create new KubeConfig(experiment)
+  delete(d)      Delete the specified context from the kubeconfig
+  help           Help about any command
+  list(ls,l)     List KubeConfig
+  merge(m)       Merge multiple kubeconfig files into one
+  namespace(ns)  Switch or change namespace interactively
+  rename(r)      Rename the contexts of kubeconfig
+  switch(s)      Switch Kube Context interactively
+  version(v)     Print version info
```

Comment:
I don't think it's a good choice to write the alias in help, it makes the output messy.

Are there other projects that do this?

Reply:
OK, updated. PTAL thanks.